strings/strings.json

```diff
@@ -7600,5 +7600,9 @@
   "UsefulnessFeedbackReason": {
     "en": "Reason",
     "fr": "Raison"
+  },
+  "AnswerSnippet": {
+    "en": "Answer to your question",
+    "fr": "Réponse à votre question"
   }
```

Comment:
@fbeaudoincoveo This string is read vocally by accessibility tools when users select the answer snippet.

Reply:
LGTM!